### PR TITLE
feat/add-gateway-arweave

### PR DESCRIPTION
**Pull request by Kunkka0822**

### CODE REVIEW

File: src/schema.mjs [hunk 1]
```diff
@@ -172,10 +172,14 @@ export const arweave = {
         },
         uri: {
           type: "string",
-          pattern: "ar://[a-zA-Z0-9-_]{43}.*",
+          pattern: "ar:\/\/[a-zA-Z0-9-_]{43}.*"
         },
+        gateway: {
+          type: "string",
+          pattern: "^https?:\/\/[a-zA-Z0-9-.]{0,}(arweave)[a-zA-Z0-9-.]{0,}\/",
```
Comment (TimDaub) [inline src/schema.mjs:179]: instead, use the "format": "uri" option

Comment (TimDaub) [reply src/schema.mjs:179]: https://github.com/neume-network/schema/blob/main/src/schema.mjs#L344

Comment (Kunkka0822) [reply src/schema.mjs:179]: What do you mean?

File: test/schema_test.mjs [hunk 2]
```diff
@@ -398,3 +399,19 @@ test("should be a valid arweave message", async (t) => {
   const valid = check(message);
   t.true(valid);
 });
+
+test("arweave gateway should be a https url", async (t) => {
+  const check = ajv.compile(arweave);
+  const message = {
+    options: {
+      uri: "ar://ltmVC0dpe7_KxFHj0-S7mdvXSfmcJOec4_OfjwSzLRk/1",
+      gateway: "arweave.net/"
```
Comment (TimDaub) [inline test/schema_test.mjs:408]: mhh very interesting that format: uri allows this, as it should mandate `https://`. I'll take this PR over and do some experiments

**Review by TimDaub — APPROVED** [review]
great, thanks!

**Pull request merged**